app/services/section/remove_service.rb

```diff
@@ -9,18 +9,29 @@ def initialize(user:, manual_id:, section_uuid:, attributes:)
   end
 
   def call
+    begin
+      manual = Manual.find(manual_id, user)
+    rescue KeyError
+      raise ManualNotFoundError.new(manual_id)
```

Comment:
Possibly out-of-scope: I don't think `Manual.find` raises a `KeyError` any more; I think it raises a `Manual::NotFoundError` instead. Thus we could probably ditch this `rescue` block and rescue the latter exception in `ManualWithdrawer#execute`.